truewiki/views/edit.py

```diff
@@ -125,4 +125,5 @@ def view(user, page: str) -> web.Response:
         )
 
     body = source.create_body(wiki_page, user, "Edit", new_page=page)
+
```

Comment:
If you remove this newline, you edit one less file :) Which is always a good thing ;)